index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="css/style.css" type="text/css">
+    <title> aloha-desktop-1240px.png</title>
+  </head>
+  <body>
+    <div class="container">
+    <header>
+      <img src="images/aloha-logo.svg" alt="aloha image" id="aloha">
+      <nav>
+      <ul>
+        <li><a href="#">ABOUT</a></li>
```

Comment:
Use CSS text transform (uppercase) anywhere you have all caps letters. Do not capitalize in HTML.